CODEOWNERS

```diff
@@ -10,18 +10,18 @@
 
 /packages/integration-tests @vdovhanych @mroz22
 
-/packages/rollout @mroz22 @goodhoko
+/packages/rollout @mroz22
 
 /packages/suite @szymonlesisz @tomasklim @mroz22
 
-/packages/suite-data @tomasklim @goodhoko
+/packages/suite-data @tomasklim
 
 /packages/suite-desktop @tomasklim @szymonlesisz
 
 /packages/suite-native @matejkriz
 
-/packages/suite-storage @goodhoko
+/packages/suite-storage
```

Comment:
We probably shouldn't leave this unassigned. I just wasn't sure who to assign. .)